config/samples/v1beta1_eventtrigger.yaml

```diff
@@ -1,12 +1,6 @@
 apiVersion: vertica.com/v1beta1
 kind: EventTrigger
 metadata:
-  labels:
```

Comment:
Should we remove these labels from other samples?

Reply:
There are no labels left in other sample configs